src/main/java/uk/gov/justice/digital/delius/controller/secure/UserController.java

```diff
@@ -28,18 +27,17 @@
 
     private final UserService userService;
 
-    @ApiOperation(value = "Returns a map of user details for supplied usernames - POST version to allow large user lists.", notes = "user details for supplied usernames",
-                  nickname = "getUserDetailsMap")
+    @ApiOperation(value = "Returns a list of user details for supplied usernames - POST version to allow large user lists.", notes = "user details for supplied usernames",
+                  nickname = "getUserDetailsList")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "OK", response = StaffDetails.class, responseContainer = "List"),
+            @ApiResponse(code = 200, message = "OK", response = UserDetails.class, responseContainer = "List"),
             @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
             @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
             @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
             @ApiResponse(code = 404, message = "Not found", response = ErrorResponse.class),
             @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)})
     @PostMapping(path="/users/list/detail", consumes = "application/json")
-    public Map<String, Optional<UserDetails>> getUserDetailsMap(final @RequestBody Set<String> usernames){
-        log.info("getUserDetailsMap called with {}", usernames);
-        return userService.getUserDetailsMap(usernames);
+    public List<Optional<UserDetails>> getUserDetailsList(final @RequestBody Set<String> usernames){
```

Comment:
don't think it makes much sense for this to be `Optional` and should have a wrapper object too - see https://mojdt.slack.com/archives/GCBGDGCLB/p1565767282002300?thread_ts=1565679964.000300&cid=GCBGDGCLB

Reply:
thanks and agreed, removed the Optional and wrapped the list in a object